git/README.md

```diff
@@ -0,0 +1,44 @@
+# Git
+
+Git is pretty awesome. Here's some reasonable standards when it comes to using it.
+
+## Commit Messages
+
+At Mobify, we recommend a few standards regarding git commit messages:
+- If the commit is related to a JIRA ticket, prefix the case-sensitive ticket number to the start of the message
```

Comment:
A couple thoughts:
- JIRA + GitHub integration requires that git commit messages start with a case-sensitive JIRA ticket number in order to work.
- Not all commits have adequate comments in their code changes. The JIRA ticket in the commit message helps to build the paper trail for getting context to a particular change.